agent_issuance/src/state.rs

```diff
@@ -216,47 +216,52 @@ pub async fn update_credential_configurations(state: &IssuanceState) -> anyhow::
-                                "name": "First Name",
-                                "locale": "en"
-                            }],
-                        },
-                        {
-                            "path": ["credentialSubject", "last_name"],
-                            "display": [{
-                                "name": "Last Name",
-                                "locale": "en"
-                            }],
-                        },
-                        {
-                            "path": ["credentialSubject", "dob"],
-                            "display": [{
-                                "name": "Date of Birth",
-                                "locale": "en"
-                            }],
-                        }
-                    ]
+                          {
+                            "credential_configuration_id": "001",
+                            "format": "jwt_vc_json",
+                            "credential_definition": {
+                              "type": ["VerifiableCredential"]
+                            },
+                            "display": [
+                              {
+                                "name": "Verifiable Credential",
+                                "locale": "en",
+                                "logo": {
+                                  "uri": "https://www.impierce.com/external/impierce-logo.png",
+                                  "alt_text": "Impierce Logo"
+                                }
+                              }
+                            ],
+                            "claims": [
+                                {
+                                    "path": ["credentialSubject", "first_name"],
+                                    "display": [{
+                                        "name": "First Name",
+                                        "locale": "en"
+                                    }],
+                                },
+                                {
+                                    "path": ["credentialSubject", "last_name"],
+                                    "display": [{
+                                        "name": "Last Name",
+                                        "locale": "en"
+                                    }],
+                                },
+                                {
+                                    "path": ["credentialSubject", "dob"],
+                                    "display": [{
+                                        "name": "Date of Birth",
+                                        "locale": "en"
+                                    }],
+                                }
+                            ],
+                            "authorization_grant": {
+                                "PreAuthorized": {
+                                    "tx_code_constraints": null
                   }
-                ]))
+                }
```

Comment:
You had to add this code here because `authorization_grant` doesn't have a sensible default (which it should, see https://github.com/impierce/ssi-agent/pull/213/files#r2392404239).

After you've implemented the sensible default you should be able to revert all the changes in this JSON block